tests/suites/1_stateful/02_query/02_0006_system_table_query_speed.py

```diff
@@ -11,17 +11,22 @@
 cursor.execute("drop database if exists test;")
 cursor.execute("create database test;")
 for i in range(100):
-    sql = 'CREATE TABLE test.table_name_{i}(id int(5))'.format(i=i)
+    sql = "CREATE TABLE test.table_name_{i}(id int(5) comment 't\est', c1 int comment 'sss') comment='test'".format(i=i)
     cursor.execute(sql)
 db.commit()
 
-cursor.execute("select count() from system.tables where database='test'")
-start = datetime.now()
-cursor.execute("select count() from system.tables where database='test'")
-execute = (datetime.now()-start)
-if execute.total_seconds() > 0.05:
-    print("Err: query 100 tables in one db cost exception, cost: ", execute)
-else:
-    print("normal")
+def test_speed(cursor, name):
+    sql = "select count() from system."+ name +" where database='test'"
+    cursor.execute(sql)
+    start = datetime.now()
+    cursor.execute(sql)
+    execute = (datetime.now()-start)
+    # in ci 0.06s, in local debug build cost 0.03s
+    if execute.total_seconds() > 0.08:
```

Comment:
We only need to ensure that it is less than a certain value, for example 1s.
The test is to check if have a performance degradation, not a benchmark.

It's important to note that the CI environment may vary, which could impact the test results.

Reply:
I think it's ok? In ci it will cost 0.068s.

Reply:
It may be flaky, CI env is **vary**.

Reply:
up-threshold to 0.1 second.